js/data.js

```diff
@@ -1,23 +1,23 @@
 /* exported data */
 /* exported genres */
 
-var data = {
+let data = {
```

Comment:
This should be a const, reference data types should always be `const` because you can still add and remove values from them because they store a memory address rather then a primitive value.